### DISCUSSION TITLE
Remove alltheinternet.com from you blacklist. It shouldn't be listed.

### DISCUSSION THREAD

**Discussion by GeraldODea** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#679] [General]
All the Internet @ alltheinternet.com is a meta search engine and should not be listed on your blacklist. Not sure why it was listed, but please remove it as soon as possible. Thanks, Gerald O'Dea, Advanced Search Technologies, Inc.

**Comment by smed79** [comment]
https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/issues/680
